### ISSUE TITLE
No way of setting image resolution

### ISSUE THREAD

**Issue by ronreiter** [w3c/html-media-capture#39]
The spec today does not let users specify what resolution the image should be in. This causes low image quality in some devices.

**Comment by anssiko** [comment]
That is correct and this limitation is there by design. The capture attribute is considered a hint.

As a workaround, it is possible to check the resolution using the FileReader API along the lines of [example 6](https://w3c.github.io/html-media-capture/#example-6). Please let us know if that would work for your use case. Thanks!

**Comment by ronreiter** [comment]
So the spec will always try to, by design, take the largest image possible from the device?
